docs/README_BODY.md

```diff
@@ -237,3 +239,40 @@ trackingEvent.customField = "RandomCustomField"
 ```
 
 If you had any problem with importing the Objective-C class into Swift, please take a look at Apple's guide on [Importing Objective-C into Swift](https://developer.apple.com/documentation/swift/imported_c_and_objective-c_apis/importing_objective-c_into_swift).
+
+### Make a match
+
+Match requests are used to link multiple identifiers in Velocidi's CDP. This way, any action made with any of the identifiers, across multiple channels (Browser, Mobile App, etc..), can be associated to the same user.
+
+In VelocidiSDK, a match request will link the user's Advertising Identifier with other provided identifiers (like an internal ID). A typical use case for this is, for instance, during the login action, to associate the user's ID with Apple's Advertising Identifier (identifier used in all the tracking event requests).
```

Comment:
What do you think?

```suggestion
With this SDK, a match request will link the user's Advertising Identifier with other provided identifiers (like an internal ID). A typical use case for this is, for instance, during the login action, to associate the user's ID with Apple's Advertising Identifier (identifier used in all the tracking event requests).
```

Reply:
I'd say this is the same thing

Reply:
It is, but to me it seems less confusing, but I don't have strong feelings. If you prefer to keep as is, then please fix formatting:

```suggestion
In Velocidi SDK, a match request will link the user's Advertising Identifier with other provided identifiers (like an internal ID). A typical use case for this is, for instance, during the login action, to associate the user's ID with Apple's Advertising Identifier (identifier used in all the tracking event requests).
```

Reply:
But VelocidiSDK is the name of the framework, the formatting is not wrong 😅

Reply:
🙈